srcs/backend/accounts/templates/registration/form.html

```diff
@@ -0,0 +1,30 @@
+<section>
+	<div class="card__container">
+			<div class="card">
+					<div class="card__content">
+							<h2 class="card__title">{{ form_title }}</h2>
+							<form onsubmit=handleFormSubmit(event) action="{{ form_action }}">
+									{% csrf_token %}
+									{{ form.as_div }}
+									<input type="hidden" id="redirectUrl" name="redirect_url" value="/"/>
```

Comment:
I think this part with an empty input field hack is unnecessary, everything seems to work just fine with something like this:
`<a href="{{ alt_action_url }}" class="font-size--xs" onclick='navigate("{{ alt_action_url }}")'>{{ alt_action }}</a>`
or actually even this:
`<a href="{{ alt_action_url }}" class="font-size--xs">{{ alt_action }}</a>`
I removed everything related to it, tested and it seems to work, so I suggest to remove it and simplify the code

Reply:
I add the empty input field becuase I want the login from navbar and tournament button to behave differently. For the login from navbar, it will redirect back to the home page, while the login from tournament button will redirect to the tounament page. Once login from navbar or tournament button is clicked, the value field will change according to the parameter passed from the navigate function. And then when the form is submitted, the backend will extract the url and pass it back to the frontend to fetch the correct page.

Reply:
Ahhh I see, thanks for explaining! That makes sense then, I didn't think of this functionality. Nice hack!